### PR TITLE
wavesurfer.js: update to 5.1

### PR DESCRIPTION
**Pull request by caugner**
See: https://github.com/katspaugh/wavesurfer.js/blob/5.1.0/CHANGES.md

Changes:
- add MarkerParams.markerElement (src/plugin/markers)

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by caugner** [comment]
@sundayz @thijstriemstra The other maintainers haven't responded yet unfortunately. Would you mind being added to the @types/wavesurfer.js maintainer list, so that you can also review changes to the wavesurfer.js type definitions?

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/c476a83b11b492eff5c9fb24992eb6214146ffae/types/wavesurfer.js/index.d.ts#L1-L7

**Review by thijstriemstra — APPROVED** [review]

**Comment by thijstriemstra** [comment]
> Would you mind being added to the @types/wavesurfer.js maintainer list, so that you can also review/approve changes to the wavesurfer.js type definitions?

Sounds good.

File: types/wavesurfer.js/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for wavesurfer.js 5.0
+// Type definitions for wavesurfer.js 5.1
```
Comment (thijstriemstra) [inline types/wavesurfer.js/index.d.ts:1]: use 5.1.0 instead

Comment (caugner) [reply types/wavesurfer.js/index.d.ts:1]: Most type definitions reference the minor version, as patch versions shouldn't have interface changes in patch versions. My impression was that wavesurfer.js does adhere to semantic versioning. Does it not?

**Comment by caugner** [comment]
> Sounds good.

Done in b62bb01.

**Comment by caugner** [comment]
@higuri @kubk @bennordgengo Would you like to stay owner of the wavesurfer.js type definitions, or would you prefer to be removed from that list?

**Review by thijstriemstra — APPROVED** [review]

**Comment by caugner** [comment]
Ready to merge

**Comment by caugner** [comment]
Thanks for the review @thijstriemstra.

**Comment by thijstriemstra** [comment]
@caugner https://github.com/katspaugh/wavesurfer.js#using-with-a-module-bundler could mention types package installation (as optional of course), interesting in doing pr for that?

**Pull request merged**